Jenkinsfile

```diff
@@ -80,28 +80,6 @@ pipeline {
         }
       }
     }
-    stage('Checks'){
-      options { skipDefaultCheckout() }
-      environment {
-        GOFLAGS = '-mod=readonly'
-      }
-      steps {
-        withGithubNotify(context: "Checks") {
-          stageStatusCache(id: 'Checks'){
-            // test the ./dev-tools/run_with_go_ver used by the Unified Release process
-            dir("${BASE_DIR}") {
-              sh "HOME=${WORKSPACE} GO_VERSION=${GO_VERSION} ./dev-tools/run_with_go_ver make test-mage"
-            }
-            withBeatsEnv(archive: false, id: "checks") {
-              dumpVariables()
-              whenTrue(env.ONLY_DOCS == 'false') {
-                runChecks()
```

Comment:
Is something outside this PR taking care of this now? It's not obvious to me that the check step for each project is being run as it was before looking at just the changes here.

Reply:
Yes, GitHub actions are now in place, `runChecks` was migrated 95% to GitHub actions in:
- https://github.com/elastic/beats/pull/32723
- https://github.com/elastic/beats/pull/32749
- https://github.com/elastic/beats/pull/32711
- And the list of other beats in https://github.com/elastic/beats/pull/32711#ref-commit-1c0c6b5

The only left-over was the checks stage in `deploy/kubernetes/Jenkinsfile.yml`